src/Twig/Extension/SonataDoctrineORMAdminExtension.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Twig\Extension;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class SonataDoctrineORMAdminExtension extends AbstractExtension
```

Comment:
the class should be final

Reply:
this class will be moved to admin bundle.